Source/Core/ArcGisImageServerTerrainProvider.js

```diff
@@ -61,6 +60,7 @@ define([
      *   proxy : new Cesium.DefaultProxy('/terrain/')
      * });
      * viewer.terrainProvider = terrainProvider;
+     *  @see TerrainProvider
```

Comment:
All similiar files.  If you make a commit to the same branch, the pull request gets automatically updated.  Just comment on the pull request when this is ready for another review.